tasker/core/task_executor_main.py

```
@@ -2276,6 +2293,221 @@ def execute_task(self, task):
+
+        Uses os.kill with signal 0 (doesn't actually kill, just checks existence).
+
+        Args:
+            pid: Process ID to check
+
+        Returns:
+            bool: True if process is running, False otherwise
+        """
+        try:
+            os.kill(pid, 0)
+            return True
```

Ruff (0.14.4)

Consider moving this statement to an `else` block

(TRY300)